Resources/Prototypes/_EinsteinEngines/Entities/Modsuits/security-responsory.yml

```
@@ -0,0 +1,160 @@
+      modifiers:
+        coefficients:
+          Blunt: 0.4
+          Slash: 0.4
+          Piercing: 0.3
+          Heat: 0.5
+          Radiation: 0.25
+          Caustic: 0.4
+    - type: SealableClothingVisuals
+      visualLayers:
+        outerClothing:
+        - state: equipped-OUTERCLOTHING-sealed
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)